components/places/src/import/common.rs

```diff
@@ -16,15 +16,18 @@ pub mod sql_fns {
     #[inline(never)]
     pub fn sanitize_timestamp(ctx: &Context<'_>) -> Result<Timestamp> {
         let now = Timestamp::now();
-        Ok(if let Ok(ts) = ctx.get::<Timestamp>(0) {
-            if Timestamp::EARLIEST < ts && ts < now {
-                ts
-            } else {
-                now
+        let is_sane = |ts: Timestamp| -> bool { Timestamp::EARLIEST < ts && ts < now };
+        if let Ok(ts) = ctx.get::<Timestamp>(0) {
```

Comment:
Actually, can you read this as an i64 instead? I'm worred we might enforce the bounds in the FromSql in the future.